osr/handlers/teleports/rsteleports.simba

```diff
@@ -2,6 +2,9 @@
 {$IFNDEF WL_OSR}
   {$I WaspLib/osr.simba}
 {$ENDIF}
+{$IFNDEF SKUNK_TELEPORT_UTILS}
```

Comment:
You don't need this btw, by ensuring WL is included with the above, this rateleportutils.simba is also included